app.js

```diff
@@ -11,15 +11,16 @@ app.get("/api/users", controller.getAllUsers);
 app.patch("/api/articles/:article_id", controller.patchArticle);
 app.get("/api/articles", controller.getAllArticles);
 app.get("/api/articles/:article_id/comments", controller.getAllComments);
+app.post("/api/articles/:article_id/comments", controller.postNewComment);
 
 app.use((err, req, res, next) => {
     console.error(err);
     res.status(500).send('Server Error!');
 });
 
-// app.listen(5000, ()=>
-// {
-//     console.log("Listening on port 5000");
-// });
+app.listen(5000, ()=>
```

Comment:
You shouldn't need this yet, but remember when you do to export it to a seperate file.